databricks/koalas/tests/test_indexes.py

```diff
@@ -1052,6 +1041,64 @@ def test_monotonic(self):
                     self.assert_eq(kmidx.is_monotonic_increasing, pmidx.is_monotonic_increasing)
                     self.assert_eq(kmidx.is_monotonic_decreasing, pmidx.is_monotonic_decreasing)
 
+        # The datas below are tested another way since they cannot be an arguments for
+        # `MultiIndex.from_tuples` in pandas >= 1.1.0.
+        # Refer https://github.com/databricks/koalas/pull/1688#issuecomment-667156560 for detail.
+        if LooseVersion(pd.__version__) < LooseVersion("1.1.0"):
+            pmidx = pd.MultiIndex.from_tuples(
+                [(-5, None), (-4, None), (-3, None), (-2, None), (-1, None)]
+            )
+            kmidx = ks.from_pandas(pmidx)
+            if LooseVersion(pd.__version__) < LooseVersion("1.1.4"):
+                self.assert_eq(kmidx.is_monotonic_increasing, False)
+                self.assert_eq(kmidx.is_monotonic_decreasing, False)
+            else:
+                self.assert_eq(kmidx.is_monotonic_increasing, pmidx.is_monotonic_increasing)
+                self.assert_eq(kmidx.is_monotonic_decreasing, pmidx.is_monotonic_decreasing)
```

Comment:
We don't need to branch by `LooseVersion(pd.__version__) < LooseVersion("1.1.4")` as here is in `if LooseVersion(pd.__version__) < LooseVersion("1.1.0"):` clause.